internal/engine/experiment/torsf/torsf.go

```diff
@@ -0,0 +1,183 @@
+// Package torsf contains the torsf experiment. This experiment
+// measures the bootstrapping of tor using snowflake.
+//
+// See https://github.com/ooni/spec/blob/master/nettests/ts-030-torsf.md
+package torsf
+
+import (
+	"context"
+	"path"
+	"time"
+
+	"github.com/ooni/probe-cli/v3/internal/engine/model"
+	"github.com/ooni/probe-cli/v3/internal/engine/netx/archival"
+	"github.com/ooni/probe-cli/v3/internal/ptx"
+	"github.com/ooni/probe-cli/v3/internal/tunnel"
+)
+
+// TODO(bassosimone):
+//
+// 1. how the measure then number of bytes consumed by the bootstrap?
+//
+// 2. can we obtain the snowflake logs?
+//
+// 3. make sure we set the IsAnomaly key in the summary.
+//
+// 4. what else?
+
```

Comment:
```suggestion
```

Reply:
We now have a tracking epic for future improvements: https://github.com/ooni/probe/issues/1686